src/components/etherscan-link/etherscan-link.tsx

```diff
@@ -91,21 +91,18 @@ export const EtherscanLink = ({
     );
   };
 
```

Comment:
Why was this removed? The links are not mono any more.

Reply:
I dont think this component should define what type of font to use so I'm going to allow you to specify a classname if needed